apps/web/src/components/hero/index.tsx

```diff
@@ -13,10 +13,12 @@ function Hero(): React.JSX.Element {
   const onSubmit = (e: React.FormEvent): void => {
     e.preventDefault()
 
-    if (email === '') {
+    const emailRegex = /^[^\s@]+@[^\s@]+\.[\w]{2,}$/
```

Comment:
![image](https://github.com/user-attachments/assets/d445cbf3-8b37-45d0-8d76-1cb53ed6925b)
The pattern is still broken. Please fix it. Use regex101 to test your patterns.

Reply:
([a-zA-Z0-9]+)([\_\.\-{1}])?([a-zA-Z0-9]+)\@([a-zA-Z0-9]+)([\.])([a-zA-Z\.]+)
@rajdip-b
I think this regex will do but it is failing pre commit checks because of es-lint, it says
 "Capture group '([a-zA-Z0-9]+)' should be converted to a named or non-capturing group."
 but if i change it to name capturing then it says 
"Named capturing groups are only available when targeting 'ES2018' or later."

Reply:
@bansal-harsh-2504 you can just use email validation from zod, it will make your life easy